.github/.backend_git_ref

```diff
@@ -0,0 +1 @@
+main
```

Comment:
does this make sense? the CI run is not reproducible

Reply:
Convenient, ist es so schlimm?